go/logic/migrator.go

```diff
@@ -351,6 +351,18 @@ func (this *Migrator) Migrate() (err error) {
 	if err := this.initiateInspector(); err != nil {
 		return err
 	}
+	// In MySQL 8.0 (and possibly earlier) some DDL statements can be applied instantly.
```

Comment:
I think we should start binlog streaming before the instant DDL is attempted. This might not change integrity, but it would be more consistent with the current approach/docs

Moving this below `this.initiateStreaming()` would achieve this

Reply:
Sure, i've moved it below flag file creation as well. The only downside is the log writes more info that is unnecessary. But it's a non-default flag now, and I think that's fine.